zephyr/wrapper.c

```diff
@@ -72,6 +72,9 @@ __section(".heap_mem") static uint8_t __aligned(64) heapmem[HEAPMEM_SIZE];
 
 #else
 
+#define HEAP_SYSTEM_SHARED_SIZE	0x1500
+#define HEAP_RUNTIME_SHARED_SIZE	0x10000
+#define HEAP_BUFFER_SIZE	0x10000
```

Comment:
@lyakh why is this needed by Zephyr at all given that it has it own linker script and allocator ?

Reply:
yes, @lyakh I need your help here, as it still fails on ICL zephyr building with this hard-coding and it breaks running on device test.

Reply:
@lgirdwood with Zephyr heap is allocated in wrapper.c as an empty static array, i.e. a .bss object. That means it needs a fixed size, known at compile-time. And 64K seems way too little to me for all the allocations on all platforms.

Reply:
> @lgirdwood with Zephyr heap is allocated in wrapper.c as an empty static array, i.e. a .bss object. That means it needs a fixed size, known at compile-time. And 64K seems way too little to me for all the allocations on all platforms.

@lyakh do you propose to use different values for different platforms via "#ifdef" here?

Reply:
@keyonjie first it would be good to know what heap sizes we're currently using with Zephyr on all platforms. Maybe that information is available somewhere during linking / rimage processing or it should be easy to print it for debugging. Then an obvious safe solution would be to use the same values. But in the long term I think we should also switch over to link-time calculated heap size for Zephyr. For that we could get the linker provide a heap start and heap size address to be used in `statics_init()` in wrapper.c. I'm not a linker expert but I think it should be possible.